work_queue/src/work_queue.c

```diff
@@ -6770,6 +6784,9 @@ int work_queue_tune(struct work_queue *q, const char *name, double value)
 	} else if(!strcmp(name, "wait-for-workers")) {
 		q->wait_for_workers = MAX(0, (int)value);
 
+	} else if(!strcmp(name, "delayed-wait")){
+		q->delayed_wait = MAX(0, (int)value);
```

Comment:
Add documentation to `work_queue.h` in `work_queue_tune` and `bindings/python3/work_queue.binding.py`